packages/afl-persistent/afl-persistent.1.4/opam

```diff
@@ -9,6 +9,9 @@ build: [
   [ "dune" "build" "-p" name "-j" jobs ]
   [ "./config.sh" ]
 ]
+patches: [
+  "config.patch"
+]
```

Comment:
what is missing here is a section:
```
extra-files: [
  ["config.patch" "md5=<>"]
```

at least with the upcoming change in opam https://github.com/ocaml/opam/pull/5564 this `extra-files` section is necessary...

Reply:
I like this change, thanks for pointing it out. I'll send a PR immediately.